### DISCUSSION TITLE
How to disable GitHub pull request merge button when CI job got failed?

### DISCUSSION THREAD

**Discussion by ElangoRajendran** [community/community#21611] [New to GitHub]
<p>How to disable GitHub pull request merge button when the CI job got failed?<br>
Merge button got enable only CI job got success otherwise it should be disabled.</p>
<p>The user could not merge the pull request changes when the CI job got failed. The user can merge only CI job got success.</p>
<p>Are there any possibilities to do this?</p>
<p>Could you suggest a solution for this?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>You can do that with branch protection rules, see the documentation here: <a href="https://docs.github.com/en/github/administering-a-repository/about-protected-branches#require-status-checks-before-merging">Require status checks before merging</a></p>

**Comment by ElangoRajendran** [comment]
<p>Hi <a class="mention" href="/airtower-luna">@airtower-luna</a></p>
<p>Thanks for the update, it’s working fine.</p>
<p>After enabling the Require status checks before merging, I have added value as build on Status checks that are required column, it is working fine.</p>
<p>Regards,<br>
Elango R.</p>

**Reply by QinisoM** [reply]
Does this work on a free account/tier?
